frames/src/main/java/com/checkout/frames/cvvcomponent/api/CVVComponentApiClient.kt

```diff
@@ -0,0 +1,17 @@
+package com.checkout.frames.cvvcomponent.api
+
+import android.content.Context
+import com.checkout.base.model.Environment
+import com.checkout.frames.cvvcomponent.models.CVVComponentConfig
+
+@Suppress("UnusedPrivateMember")
+internal class CVVComponentApiClient(
+    private val publicKey: String,
+    private val environment: Environment,
+    private val context: Context,
+) : CVVComponentApi {
+
+    override fun createComponentMediator(cvvComponentConfig: CVVComponentConfig): ComponentMediator {
+        return InternalComponentMediator(cvvComponentConfig)
+    }
+}
```

Comment:
Curious why we take param without using them?

Reply:
It is a further work in progress. We will use it in further following PRs. It will used for Logger setup and tokenisation in ComponentMediator